core/src/test/resources/logback.xml

```diff
@@ -0,0 +1,11 @@
+<configuration debug="true">
```

Comment:
Now that we're using `logback` and a proper logging setup during testing, this `logback.xml` ensures logging is configured to hide as much of the stuff as possible (because otherwise it's insanely noisy).